app/main.py

```diff
@@ -1 +1,7 @@
-# write your code here
+def copy_file(command: str) -> None:
+    temp_arr = command.split()
+    if len(temp_arr) == 3 and temp_arr[1] != temp_arr[2]:
+        with (open(temp_arr[1], "r") as source,
+              open(temp_arr[2], "w") as copied_file
+              ):
```

Comment:
In case you have arguments on the same line as the opening parenthesis, you should place the closing parenthesis on the line with arguments as well. Reformat your code to encounter that.